terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,84 @@
+  docker_repo               = "presenter-account-web"
+  lb_listener_rule_priority = 16
+  lb_listener_paths         = ["/presenter-account/*"]
+  healthcheck_path          = "/presenter-account/healthcheck" #healthcheck path for presenter-account-web
+  healthcheck_matcher       = "200"                        # no explicit healthcheck in this service yet, change this when added!
+
+  kms_alias       = "alias/${var.aws_profile}/environment-services-kms"
+  service_secrets = jsondecode(data.vault_generic_secret.service_secrets.data_json)
+
+  parameter_store_secrets = {
+    "vpc_name"             = local.service_secrets["vpc_name"]
+    "chs_api_key"          = local.service_secrets["chs_api_key"]
+    "internal_api_url"     = local.service_secrets["internal_api_url"]
+    "chs_internal_api_key" = local.service_secrets["chs_internal_api_key"]
+    "cache_server"         = local.service_secrets["cache_server"]
+    "account_url"          = local.service_secrets["account_url"]
+    "cookie_secret"        = local.service_secrets["cookie_secret"]
+    "oauth2_auth_uri"      = local.service_secrets["oauth2_auth_uri"]
+    "oauth2_redirect_uri"  = local.service_secrets["oauth2_redirect_uri"]
+    "oauth2_token_uri"     = local.service_secrets["oauth2_token_uri"]
+    "oauth2_client_id"     = local.service_secrets["oauth2_client_id"]
+    "oauth2_client_secret" = local.service_secrets["oauth2_client_secret"]
+    "oauth2_request_key"   = local.service_secrets["oauth2_request_key"]
+  }
+
+  vpc_name             = local.service_secrets["vpc_name"]
+  chs_api_key          = local.service_secrets["chs_api_key"]
+  internal_api_url     = local.service_secrets["internal_api_url"]
+  chs_internal_api_key = local.service_secrets["chs_internal_api_key"]
+  cache_server         = local.service_secrets["cache_server"]
+  account_url          = local.service_secrets["account_url"]
+  cookie_secret        = local.service_secrets["cookie_secret"]
+  oauth2_auth_uri      = local.service_secrets["oauth2_auth_uri"]
+  oauth2_redirect_uri  = local.service_secrets["oauth2_redirect_uri"]
+  oauth2_token_uri     = local.service_secrets["oauth2_token_uri"]
+  oauth2_client_id     = local.service_secrets["oauth2_client_id"]
+  oauth2_client_secret = local.service_secrets["oauth2_client_secret"]
+  oauth2_request_key   = local.service_secrets["oauth2_request_key"]
+
+  # create a map of secret name => secret arn to pass into ecs service module
+  # using the trimprefix function to remove the prefixed path from the secret name
+  secrets_arn_map = {
+    for sec in data.aws_ssm_parameter.secret :
+    trimprefix(sec.name, "/${local.name_prefix}/") => sec.arn
+  }
+
+  service_secrets_arn_map = {
+    for sec in module.secrets.secrets :
+    trimprefix(sec.name, "/${local.service_name}-${var.environment}/") => sec.arn
+  }
+
+  task_secrets = [
```

Comment:
You don't need to list all the secrets and environment variables here anymore you can add them programmatically like this: https://github.com/companieshouse/presenter-account-consumer/blob/main/terraform/groups/ecs-service/locals.tf